### DISCUSSION TITLE
Links to help changed after Pi.Alert URL changed to Local

### DISCUSSION THREAD

**Discussion by masterwishx** [netalertx/NetAlertX#166] [Q&A]
All links that points to github Help pages changed aftre i changed Pi.Alert URL to http://192.168.0.199:20211

so instead https://pi.alert.com..../settings.php#SCAN_SUBNETS that point to help
i have : http://192.168.0.199:20211/settings.php#SCAN_SUBNETS

![image](https://user-images.githubusercontent.com/28630321/216701740-8b5612e3-f83d-4643-a6f2-9e51f84b8ce0.png)

**Comment by jokob-sk** [comment]
> i have : http://192.168.0.199:20211/settings.php#SCAN_SUBNETS

That's the correct behavior. These links point to the specific settings within this settings page. These links never pointed to anywhere on github else before. All links ending with `#SETTING_NAME` point to a specific section (setting) within your current settings page.

**Reply by masterwishx** [reply]
Sorry, now I can see I think I confused with other links that points to help pages
